websites/G/Granblue Fantasy Wiki/dist/metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.5",
+  "author": {
+    "name": "Kyrie",
+    "id": "368399721494216706"
+  },
+  "service": "Granblue Fantasy Wiki",
+  "description": {
+    "en": "Granblue Fantasy Wiki"
+  },
+  "url": "gbf.wiki",
+  "version": "1.0.0",
+  "logo": "https://i.imgur.com/k5QiAPC.png",
+  "thumbnail": "https://i.imgur.com/3vIpCD7.png",
+  "color": "#2F4F7A",
+  "tags": [
+    "granblue",
+    "fantasy",
+    "gbf",
+    "wiki"
+  ],
+  "category": "anime"
```

Comment:
Hmm anime?

Reply:
Considering I based this of from Bas950's _Azur Lane Wiki_ and he wrote it down as "anime",and that both games shares the same genre.  
So yeah, anime

Reply:
@Slowlife01 what category do you think is better for this presence and I'll fix it ASAP

Reply:
Changed it to `game`, minor mistake sorry